src/OptimalTransport.jl

```diff
@@ -117,62 +117,114 @@ function emd2(μ, ν, C, optimizer; plan=nothing)
 end
 
 """
-    sinkhorn_gibbs(mu, nu, K; tol=1e-9, check_marginal_step=10, maxiter=1000)
+    sinkhorn_gibbs(
+        μ, ν, K; atol=0, rtol=atol > 0 ? 0 : √eps, check_convergence=10, maxiter=1_000
+    )
 
-Compute dual potentials `u` and `v` for histograms `mu` and `nu` and Gibbs kernel `K` using
-the Sinkhorn algorithm (Peyre et al., 2019)
+Compute the dual potentials for the entropic regularization optimal transport problem
+with source and target marginals `μ` and `ν` and Gibbs kernel `K` using the Sinkhorn
+algorithm.
 
-The Gibbs kernel `K` is given by `K = exp.(- C / eps)` where `C` is the cost matrix and
-`eps` the entropic regularization parameter. The optimal transport plan for histograms `u`
-and `v` and cost matrix `C` with regularization parameter `eps` can be computed as
-`Diagonal(u) * K * Diagonal(v)`.
+The Gibbs kernel `K` is defined as
+```math
+K = \\exp(-C / \\varepsilon),
+```
+where ``C`` is the cost matrix and ``\\varepsilon`` the entropic regularization parameter.
+The corresponding optimal transport plan can be computed from the dual potentials ``u``
+and ``v`` as
+```math
+\\gamma = \\operatorname{diag}(u) K \\operatorname{diag}(v).
+```
+
+Every `check_convergence` steps a convergence check of the error of the marginal
+`μ` with absolute tolerance `atol` and relative tolerance `rtol` is performed. The default
+`rtol` depends on the types of `μ`, `ν`, and `K`. After `maxiter` iterations, the
+computation is stopped.
```

Comment:
Hmm, I'd like to avoid discussing internal variables of the algorithm. It's the same principle as in `isapprox`, maybe it is sufficient to refer to it? Or would you prefer a more detailed explanation? The docstrings already feel a bit (too) long but maybe this is mainly since some of the information is redundant and there are too many different functions.

Reply:
Personally, as a user, I'd prefer to have it clearly in the docstring. But I do understand if you rather leave it out.  Yes, I agree that perhaps we have too much redundant information.

Reply:
I also added some more explanations here.